### DISCUSSION TITLE
Cannot modify handlebar templates under Managed Card Fields on second panel

### DISCUSSION THREAD

**Discussion by calebmil** [microsoft-search/pnp-modern-search#3403] [Q&A]
The following scenario was found while attempting to configure a search results web part to use Modified Date and Modified By on the default results card.  Web part is version 4.7.0.

As an administrator, I wanted to modify the fields displayed on the results card in the search results web part.

1. I added ModifiedBy and ModifiedOWSDate to the list of selected properties.
2. I clicked on the Customize button under Edit layout slots for this data source.
3. I created a new slot called ModifiedBy and mapped it to ModifiedBy.
4. I created a new slot called ModifiedDate and mapped it to ModifiedOWSDate.
5. I clicked the save button.
6. I went back in to the Customize slots interface and confirmed that the new slots were created.
7. I then went to the second panel and clicked on the Manage card fields button.
8. I looked for a way to add a new field to the result card but did not see any controls that would allow me to do so.
9. I attempted to edit the handlebars expression for the Author field.  It permitted me update the handlebars expression to use root.slots.ModifiedBy instead of root.slots.Author.
10. When I clicked save, no changes were reflected on the search results.
11. When I clicked on the Manage card fields button, the handlebars expression had reverted to root.slots.Author.  Steps 9-11 were true when I attempted to change the handlebars expression for Date to use root.slots.ModifiedDate too.

**Comment by kasperbolarsen** [comment]
Are you referring to LastModifiedTime?

**Comment by calebmil** [comment]
Perhaps.  My managed properties list has the field listed as ModifiedOWSDate.  

I hacked around the issue by deleting the slot definition for the Author and Date slots and redefined those slot names to point to the fields I wanted to have displayed but the problem is that Manage card fields interface does not commit the changed handlebar template when I save it.  It also does not allow me to change the slot I want to have displayed on the card.

**Comment by wobba** [comment]
Hi,
the slots on the card is set. What you do is change the Value field to point to your new slot/value - in the position on the card.

To change the title, then something like this:

<img width="925" alt="image" src="https://github.com/microsoft-search/pnp-modern-search/assets/51104/4cfb1a6a-6058-41c5-9e6e-46286657db78">

Does this work for you? As you cannot add new UX slots to the card - it is fixed and you change the markup/value per UX slot.
